src/ActionPanel.js

```diff
@@ -46,25 +46,33 @@ class ActionPanel extends React.Component<ActionPanelProps, {}> {
             ariaLabelObj['selectedCommandName'] =
                 this.props.intl.formatMessage(
                     { id: 'ActionPanel.selectedCommandName' },
-                    { selectedCommandName: this.props.selectedCommandName }
+                    { selectedCommandName:
+                        this.props.intl.formatMessage({id: `Command.${this.props.selectedCommandName}`})
+                    }
                 );
         }
 
         if (this.props.pressedStepIndex > 0) {
             const prevStepName = this.props.program[this.props.pressedStepIndex - 1];
             ariaLabelObj['previousStepInfo'] =
                 this.props.intl.formatMessage(
-                    { id: `CommandInfo.previousStep.${prevStepName}`},
-                    { previousStepNumber: this.props.pressedStepIndex }
+                    { id: `CommandInfo.previousStep.${prevStepName.split(/([0-9]+)/)[0]}`},
```

Comment:
We don't need to do this `split()` as all the "CommandInfo.previousStep.\*" messages are the same. See the note under `messages.json`.